packaging/arch/PKGBUILD

```diff
@@ -137,21 +147,23 @@ package() {
      SYSTEMDSYSTEMUNITDIR=/usr/lib/systemd/system \
      SNAP_MOUNT_DIR=/var/lib/snapd/snap \
      DESTDIR="$pkgdir"
+  # no tweaks for sudo are needed
+  rm -rfv "$pkgdir/etc/sudoers.d"
```

Comment:
Tabs/spaces

Reply:
I've checked with whitespace-mode and it looks ok to me. It may be slightly misleading because the lines above are a continuation of the make command and are indented one level deeper.